src/js/app.js

```diff
@@ -108,20 +108,52 @@ async function getTotal(resources, prices, movrPrice){
 
 /**
  * Make a date more human-readable
- * @param {Date} date: the date that needs to be prettified (yyyy-mm-dd format)
+ * @param {Date} date: the date that needs to be prettified (3 April 2022 format)
  * @returns {String}
  */
 function formatDate(date){
   const options = {year: 'numeric', month: 'long', day: 'numeric'}
   return new Date(date).toLocaleDateString("en-GB", options)
 }
 
+/**
+ * Format date for api request
+ * @param {Date} date: the date that needs to be prettified (yyyy-mm-dd format)
+ * @returns {String}
+ */
+function formatDateApi(date){
+  return new Date(date).toISOString().split('T')[0]
+}
+
+
+/**
+ * Returns 
+ * @returns {number[]}
+ */
+function carnageDates() {
+  //2022-04-03 first date that carnage api has
+  const carnageStartTime = 1648947601000;
+  let carnageTime = carnageStartTime
+  const nowTime = new Date().getTime()
+  const dates = [carnageTime];
+  while(true){
+    carnageTime = carnageTime + 1000 * 60 * 60 * 24 * 7
+
+    if(carnageTime > nowTime){
+      break
+    }
+    dates.push(carnageTime)
+  }
+  return dates.reverse()
+}
```

Comment:
Carnage API is updated manually, it can take up to 24 hours for it to be available.